src/meson.build

```diff
@@ -0,0 +1,117 @@
+project('bpp', ['cpp','c'],
```

Comment:
Meson flattens this in argument parsing and interprets it as `project('bpp', 'cpp','c',`

Reply:
In this case the project doesn't use `cpp`, so I'm just going to remove that.